lib/services/appmanager-service.ts

```diff
@@ -71,8 +70,8 @@ class AppManagerService implements IAppManagerService {
 			this.$loginManager.ensureLoggedIn().wait();
 
 			platforms = _.map(platforms, platform => this.$mobileHelper.normalizePlatformName(platform));
-			var cachedOptionsRelease = commonOptions.release;
-			commonOptions.release = true;
+			var cachedOptionsRelease = options.release;
```

Comment:
This code is conceptually wrong. It may be left for the hotfix release but it must be rewritten correctly in master.